### PR TITLE
Add styling to review section

### PR DESCRIPTION
**Pull request by shandyk18**
Styling was added with Material UI. 

Users can rate with the star system and optionally leave a text review.

Screenshot 2020-07-22 at 12 23 15 PM - Display 2

### CODE REVIEW

File: ubg/src/Reviews/AllReviews.js [hunk 1]
```diff
@@ -21,13 +23,18 @@ class AllReviews extends React.Component {
...
         return (
             <div className='reviews'>
-                <NewReview name={this.state.user} handleAddReview={this.handleAddReview}/>
-                <Reviews reviews={this.state.reviews}/>
+                <Box container="true" justify="center" alignItems="center" m={10}>
+                    <Typography variant="h3">
+                        Reviews
+                    </Typography>
+                    <NewReview name={this.state.user} handleAddReview={this.handleAddReview}/>
+                    <Reviews reviews={this.state.reviews} bggReviews={this.bggReviews}/>
```
Comment (danieltre23) [inline ubg/src/Reviews/AllReviews.js:36]: nit: Missing spaces before /> in NewReview and Reviews

**Comment by zhangshyue** [comment]
One thing I noticed when looking at BGG is that the review system seems to be more complicated: https://www.boardgamegeek.com/boardgame/174430/gloomhaven/ratings?rated=1&comment=1

File: ubg/src/Reviews/Review.js [hunk 2]
```diff
@@ -1,13 +1,48 @@
 import React from 'react';
+import Divider from "@material-ui/core/Divider";
```
Comment (zhangshyue) [inline ubg/src/Reviews/Review.js:2]: Maybe be consistent on how you import components from material-ui

File: ubg/src/Reviews/NewReview.js [hunk 3]
```diff
@@ -1,4 +1,6 @@
 import React from 'react';
+import { Box, Button, InputLabel, TextField } from '@material-ui/core/';
```
Comment (zhangshyue) [inline ubg/src/Reviews/NewReview.js:2]: Shall we use the second way to import components

**Pull request merged**